File: subsetand.py
t=input()
while(t):
	t=t-1
	x=raw_input()
	z,n=map(int,x.split())
	a=[]
	x=raw_input()
	a=map(int,x.split())
	l=len(a)
	b=[]
	for i in range(0,l):
		b.append(z&a[i])
	d=0
	for i in range(0,l):
		if b[i]==0:
			d=1
			break
	if d==1:
		print "Yes"
	else:
		print "No"

File: subsetand.c
#include<stdio.h>
main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int z,n,i;
        scanf("%d %d",&z,&n);
        int a[n],b[n];
        for(i=0;i<n;i++)
        {scanf("%d",&a[i]);
        b[i]=a[i]&z;}
        for(i=0;i<n;i++)
        if(b[i]==0)
        break;
        if(i<n)
        printf("Yes\n");
        else
        printf("No\n");
    }
    return 0;
}
